docs/croissant-spec.md

```diff
@@ -171,27 +160,44 @@ The information to extract from the file. It can be: `filename`, `fullpath`,
     name is `metadata.csv`.
 *   `content`: The byte content of the file.
 
-**subclassOf**:	[sc:Text](https://schema.org/Text)
+**range**:	[sc:Text](https://schema.org/Text)
+
+### csvcolumn
+
+The column of the source CSV (e.g., `date`).
+
+**range**:	[sc:Text](https://schema.org/Text)
+
+### jsonPath
+
+The JSON path if the source is a JSON (e.g., `/bookstore/book[1]/title`).
```

Comment:
I'm not sure I understand this, but it is probably clear for someone with more context?

Reply:
Done.